bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -153,6 +156,8 @@
 
     private boolean removeTrailingSlash;
 
+    private List<String> template;
```

Comment:
```suggestion
    private List<String> excludedPageTemplates;
```
Better to be clear what this variable represents.

Reply:
Yes, its done now